todo/todo.go

```diff
@@ -85,10 +77,11 @@ func ShowAll() error {
 }
 
 func Update(task_id int, task string) error {
-	db := dbConn(user, password, dbname)
+	// db := dbConn(user, password, dbname)
+	db := db.InitDB()
```

Comment:
Why are you initialising the DB again and again. We did this once in `app.Init()` right?

Reply:
Line 90 `row.Scan(&task_id)` Wy do we need this?

Reply:
This was required for error check. Wasn't there in this version of code. Is presented in updated code now.